support/doc/api/embeds.md

```diff
@@ -58,102 +58,122 @@ For example `https://my-instance.example.com/videos/embed/52a10666-3a18-4e73-93d
 
 Start the video at a specific time.
 Value must be raw seconds or a duration (`3m4s`)
+Default: starts at `0`
 
 ### stop
 
 Stop the video at a specific time.
 Value must be raw seconds or a duration (`54s`)
+Default: ends at content end
 
 ### controls
 
 Mimics video HTML element `controls` attribute, meaning that all controls (including big play button, control bar, etc.) will be removed.
 It can be useful if you want to have a full control of the PeerTube player.
 
 Value must be `0` or `1`.
+Default: `1`
 
 ### controlBar
 
 Hide control bar when the video is played.
 
 Value must be `0` or `1`.
+Default: `1`
 
 ### peertubeLink
 
 Hide PeerTube instance link in control bar.
 
 Value must be `0` or `1`.
+Default: `1`
 
 ### muted
 
 Mute the video by default.
 
 Value must be `0` or `1`.
+Default: `0`
 
 ### loop
 
 Automatically start again the video when it ends.
 
 Value must be `0` or `1`.
+Default: `0`
 
 ### subtitle
 
 Auto select a subtitle by default.
 
 Value must be a valid subtitle ISO code (`fr`, `en`, etc.).
+Default: no subtitle selected
 
 ### autoplay
 
 Try to automatically play the video.
 Most web browsers disable video autoplay if the user did not interact with the video. You can try to bypass this limitation by muting the video
 
 Value must be `0` or `1`.
+Default: `0`
 
 ### playbackRate
 
 Force the default playback rate (`0.75`, `1.5` etc).
+Default: `1`
 
 ### title
 
-Hide embed title.
+Show embed title.
 
 Value must be `0` or `1`.
+Default: `1`
 
 ### warningTitle
 
-Hide P2P warning title.
+Show P2P warning title.
 
 Value must be `0` or `1`.
+Default: `1`
 
 ### p2p
 
-Disable P2P.
+Enable P2P.
 
 Value must be `0` or `1`.
+Default: `1`
 
 ### bigPlayBackgroundColor
 
-Customize big play button background color.
+Customize settings menu background color.
 
 Value must be a valid color (`red` or `rgba(100, 100, 100, 0.5)`).
+Default: ???
 
 ### foregroundColor
 
 Customize embed font color.
 
 Value must be a valid color (`red` or `rgba(100, 100, 100, 0.5)`).
 
+Default: 'white'
+
 ### mode
 
 Force a specific player engine.
 
 Value must be a valid mode (`web-video` or `p2p-media-loader`).
 
+Default: FIXME
```

Comment:
Addtionnally a pointer to the doc explaining what it really does would be usefull